src/main/java/com/casper/sdk/identifier/entity/PublicKeyEntityIdentifier.java

```diff
@@ -0,0 +1,27 @@
+package com.casper.sdk.identifier.entity;
+
+import com.casper.sdk.model.key.PublicKey;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * The public key of an account.
+ *
+ * @author carl@stormeye.co.uk
+ */
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+public class PublicKeyEntityIdentifier implements EntityIdentifier{
+
+    /**
```

Comment:
Use single-line JavaDoc if text does not span more than one line `/** Public key hash */`

Same applies to all new classes.